plugins/modules/swim_intent.py

```diff
@@ -50,26 +50,26 @@
         type: dict
         suboptions:
           type:
-            description: The source of import, supports url import or local import.
+            description: The source of import, supports remote import or local import.
             type: str
           local_image_details:
             description: Details of the local path of the image to be imported.
             type: dict
             suboptions:
               file_path:
-                description: File absolute path.
+                description: Give the file absolute path required while importing image from local.
                 type: str
               is_third_party:
-                description: IsThirdParty query parameter. Third party Image check.
+                description: IsThirdParty query parameter. Third party Image check (Optional).
```

Comment:
description: Query parameter to determine if the image is from a third party (optional).

Reply:
added.